aamt_parse2.pm

```diff
@@ -4713,12 +4960,13 @@ sub pars_siteHasValidFrameworkDb
     if (!$array[$sIndex][MYSQL])
     {
         my $strYesOrNo =" ";
-        while($strYesOrNo!~/^\s*[YynN]\s*$/)
+#        while($strYesOrNo!~/^\s*[YynN]\s*$/)
         {
-            ilog_printf(1, "\n    $framework site detected, would you like to automatically create and migrate the database [$strSiteName]? (Y/N):");
```

Comment:
You need to add a cmd line flag for running in unattended/non-interactive mode. If this flag is true, then you can skip this, if it is false, then keep the current code.